src/network/clientpackethandler.cpp

```diff
@@ -1009,7 +1012,7 @@ void Client::handleCommand_AddParticleSpawner(NetworkPacket* pkt)
 	event.add_particlespawner.attached_id        = attached_id;
 	event.add_particlespawner.vertical           = vertical;
 	event.add_particlespawner.texture            = new std::string(texture);
```

Comment:
I know you didn't change this, but does the `std::string` need the `new` keyword?